### ISSUE TITLE
Naturals.is_subset(Interval(0, oo)) return False

### ISSUE THREAD

**Issue by ninely** [sympy/sympy#19416]
Is this correct ?
```
from sympy.sets import Interval
from sympy import oo, Naturals, Reals

result = Naturals.is_subset(Interval(0, oo))
print(result)    # output: False, I expect it to be True
```

**Comment by oscarbenjamin** [comment]
Thanks for reporting this. It has been fixed since sympy 1.5.1. On master (and in 1.6) this gives True.

**Comment by ninely** [comment]
Thanks
